### DISCUSSION TITLE
Simulation of Pole Figures

### DISCUSSION THREAD

**Discussion by will-colea** [mtex-toolbox/mtex#1095] [Ask Anything]
Hello - 

I am new to mtex. I am trying to get comfortable with simulating pole figures. I know there is an example here (https://mtex-toolbox.github.io/PoleFigureSimulation.html), but I was wondering if someone had any other more simple examples, especially with defining the odf. Thank you for your help and time.

**Comment by kilir** [comment]
Hi,
have a look here
https://mtex-toolbox.github.io/UnimodalODFs.html
https://mtex-toolbox.github.io/FibreODFs.html
https://mtex-toolbox.github.io/BinghamODFs.html
and also have a look at the help of the functions which you can use (and combine) to simulate a wide range of odfs.

Cheers,
Rüdiger
